packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -494,7 +596,8 @@ class TreeTableState<T extends TreeNode<T>> extends State<TreeTable<T>>
         newSelectedNodeIndex = max(selectedNodeIndex - 1, 0);
         break;
       case ScrollKind.parent:
-        newSelectedNodeIndex = max(items.indexOf(selectedNode.parent), 0);
+        newSelectedNodeIndex =
+            max(items.indexOf(selectionValue.node.parent), 0);
```

Comment:
respect `scrollIntoView` in this method. I think you should either default it to true, or else you have to set it to true whenever the selection changes.

Reply:
done always true.